app/src/forms/auth/middleware/userAccess.js

```diff
@@ -97,17 +70,17 @@ const _getForm = async (currentUser, formId, includeDeleted) => {
 const currentUser = async (req, _res, next) => {
   try {
     // Validate bearer tokens before anything else - failure means no access.
-    const bearerToken = _getBearerToken(req);
+    const bearerToken = jwtService.getBearerToken(req);
     if (bearerToken) {
-      const ok = await keycloak.grantManager.validateAccessToken(bearerToken);
+      const ok = await jwtService.validateAccessToken(bearerToken);
       if (!ok) {
         throw new Problem(403, { detail: 'Authorization token is invalid.' });
```

Comment:
This is outside the scope of this change, but this should be a 401, not a 403, yes? I'm wondering if this PR is a good time to fix that, since we have to contact all our clients to tell them about the SSO change - might be a good way to get this "breaking" change done?

Reply:
Sounds good to me. Will make that change. How does this flow into documentation of the API?

Reply:
Oh boy. Good question. Maybe make the change here so that we're testing with the 401, and I'll update the docs in a separate PR since it's going to be a lot of (small) changes.

Reply:
The docs are updated, merged, and deployed. Confirmed that invalid or expired tokens are now returning 401s - hooray!